src/components/Questions/Body.vue

```diff
@@ -1,260 +1,144 @@
 <template>
-  <div
-    class="flex relative h-full overflow-y-auto"
-    :class="{ 'bg-gray-50': isPaletteVisible }"
-  >
-    <QuestionPalette
-      v-if="isPaletteVisible"
-      :hasQuizEnded="hasQuizEnded"
-      :questionSetStates="questionSetStates"
-      :currentQuestionIndex="currentQuestionIndex"
-      :title="title"
-      :subject="subject"
-      :testFormat="testFormat"
-      :maxMarks="maxMarks"
-      :numQuestions="numQuestions"
-      :quizTimeLimit="quizTimeLimit"
-      class="absolute w-full h-full sm:w-2/3 lg:w-1/2 xl:w-1/3 z-10"
-      @navigate="navigateToQuestion"
-      data-test="questionPalette"
-    >
+  <div class="flex relative h-full overflow-y-auto" :class="{ 'bg-gray-50': isPaletteVisible }">
```

Comment:
dont make changes in this file.. whatever formatting changes you made are actually hurtful.. they reduce the readability of code. please revert